R/RSocrata.R

```diff
@@ -162,15 +204,15 @@ no_deniro <- function(x) {
 #' @param email - Optional. The email to the Socrata account with read access to the dataset.
 #' @param password - Optional. The password associated with the email to the Socrata account
 #' @return httr response object
-#' @importFrom httr http_status GET content stop_for_status
+#' @importFrom httr http_status GET content stop_for_status user_agent
 #' @author Hugh J. Devlin, Ph. D. \email{Hugh.Devlin@@cityofchicago.org}
 #' @noRd
 getResponse <- function(url, email = NULL, password = NULL) {
   
   if(is.null(email) && is.null(password)){
-    response <- httr::GET(url)
+    response <- httr::GET(url, user_agent(fetch_user_agent()))
```

Comment:
Ok, agree with this. Will make the change (though strikes me as an odd style, it's semantically appropriate).

Reply:
I wish I had never suggested the namespace thing because we don't consistently follow it (there are two places outside the review that don't have it).  So, we get the downside of reduced readability without the benefit of having all foreign calls fully specified.

So I have mixed feelings about even making this comment.

Reply:
I think it makes sense, the use of `user_agent` is odd because it--in my mind--only makes sense in the context of `GET`, `POST`, `PUT`, etc instead of a standalone function. It's a good idea to call the namespace (also, I think reduces the chance of us being chided by CRAN).